api/views.py

```diff
@@ -2,28 +2,74 @@
 from . import db
 from .models import Task
 
-main = Blueprint('main',__name__)
+main = Blueprint('main', __name__)
 
+# Add task API
 @main.route('/add_task', methods=['POST'])
 def add_task():
     task_data = request.get_json()
 
     new_task = Task(
-        taskName = task_data['taskName'],
-        isCrashOut = task_data['isCrashOut']
+        taskName=task_data['taskName'],
+        isCrashOut=task_data['isCrashOut']
     )
 
     db.session.add(new_task)
     db.session.commit()
 
     return 'Done', 201
 
+
+# Update Task API
+@main.route('/update_task/<id>', methods=['PUT'])
```

Comment:
I believe it is better to group all these routes in the same "/tasks" resource. This is so it is easier to group which resources are directly involved. What do you think?